scripts/r.unpack/r.unpack.py

```diff
@@ -73,10 +73,9 @@ def main():
     os.chdir(tmp_dir)
     tar = tarfile.TarFile.open(name=input_base, mode="r")
     try:
-        data_names = [
-            tarinfo.name for tarinfo in tar.getmembers() if "/" not in tarinfo.name
-        ]
-    except:
+        data_names = [tarinfo.name for tarinfo in tar.getmembers() if "/" not in tarinfo.name]
+        # code changed here, Exception included instead of bare except
```

Comment:
Same as above. This info goes to commit messages, not code, but I appreciate the effort to comment on the code.